modules/datadog/main.tf

```diff
@@ -158,14 +163,12 @@ resource "helm_release" "datadog_agent" {
                         key: app-key
                   - name: DD_LOGS_ENABLED
                     value: "false"
-                  - name: DD_ENV
-                    value: "${var.environment}"
-                  - name: DD_CLUSTER_NAME
-                    value: "${var.cluster_name}"
-              selectors:
+                  - name: DD_TAGS
+                    value: "env:${var.environment}-${var.product_name}"
```

Comment:
Wasn't able to set more than one tag: DD_TAGS cannot be replicated, using a comma doesn't work and the two tags are concatenated with an underscore, having a space between them doesn't work either (again becomes an underscore). DD_TAGS only accepts a string, not a list